client/store/plugins/vuex-persist.js

```diff
@@ -0,0 +1,12 @@
+import VuexPersistence from 'vuex-persist';
+
+const STORAGE_KEY = process.env.VUEX_PERSIST_STORAGE_KEY;
```

Comment:
Doesn't feel like it needs rename / could be destructured. Consider renaming to something not vendor specific / simpler 💅